tasks/bake.js

```diff
@@ -483,18 +500,26 @@ module.exports = function( grunt ) {
 				if ( oldValue === undefined ) values[ forEachName ] = oldValue;
 				else delete values[ forEachName ];
 
-				return fragment;
+				content = fragment;
 
 			} else if( !forEachName ) {
 
 				processExtraBake( extraBake, filePath, destFile, values );
 
-				return linebreak + parse( includeContent, includePath, destFile, values );
+				content = linebreak + parse( includeContent, includePath, destFile, values );
 
 			} else {
 
+				content = "";
+			}
+
+			if( assign !== null ) {
+				parentValues[ assign ] = mout.string.ltrim( content );
```

Comment:
Good point. also I just realized it shouldn't actually matter. Because the code people pull in aren't intented anyway and it gets correctly intented later when they use it. So we are all good!